sandbox-web/src/main/java/com/exadel/sandbox/controllers/location_controller/CityController.java

```diff
@@ -34,21 +34,21 @@ public CityController(CityService cityService, JwtUtil jwtUtil) {
 
         final var cityResponse = cityService.create(cityRequest);
 
-        return ResponseEntity.ok(cityResponse);
+        return new ResponseEntity<>(cityResponse, HttpStatus.OK);
     }
 
     @GetMapping(produces = {"application/json"}, path = "/allActiveCities")
     public ResponseEntity<?> findCitiesByEventStatusActive() {
-        return ResponseEntity.ok()
-                .body(cityService.findCitiesByEventStatusActive());
+        return new ResponseEntity<>(cityService.findCitiesByEventStatusActive(), HttpStatus.OK);
     }
 
     @GetMapping(produces = {"application/json"}, path = "/allFavoriteCities")
     public ResponseEntity<?> findCitiesByFavoriteEvent(@RequestHeader("Authorization") AuthenticationResponse authenticationResponse) {
+
         Long userId = Long.parseLong(jwtUtil.extractUserId(authenticationResponse.getJwt().substring(7)));
```

Comment:
Better to have some util method or utility spring bean class that can extract userId. I bet this is re-usable logic in your controllers.